build.gradle

```diff
@@ -187,15 +187,34 @@ allprojects {
     maven {
       url  "https://jitpack.io"
     }
+    flatDir {
+      dirs "$rootProject.projectDir/libs"
+    }
   }
 
   apply plugin: 'java'
 
   dependencies {
+    implementation files(
+      'consumers/build/libs/civil-sdt-consumers-0.0.1-SNAPSHOT.jar',
+      'dao/build/libs/civil-sdt-dao-0.0.1-SNAPSHOT.jar',
+      'dao-api/build/libs/civil-sdt-dao-api-0.0.1-SNAPSHOT.jar',
+      'domain/build/libs/civil-sdt-domain-0.0.1-SNAPSHOT.jar',
+      'handlers/build/libs/civil-sdt-handlers-0.0.1-SNAPSHOT.jar',
+      'handlers-api/build/libs/civil-sdt-handlers-api-0.0.1-SNAPSHOT.jar',
+      'interceptors/build/libs/civil-sdt-interceptors-0.0.1-SNAPSHOT.jar',
+      'producers/build/libs/civil-sdt-producers-0.0.1-SNAPSHOT.jar',
```

Comment:
The original SDT build produces two release files: producers.war (SDT) and producers-comx.war (SDT commissioning).  The producers.war file is built from the producers project and references the other projects JAR files (consumers.jar, dao.jar etc.).  This is different in that it builds one archive that references all the project JARs including producers.  Will this work?

Reply:
Good query. Not sure but I don't see why not - all the classes are available while still within their own module jar. Unfortunately we don't have anything that can run. just yet. Producers commissioning hasn't been looked at (yet).

Reply:
OK.  We'll go with what we've got for the moment and revisit it later should the need arise.